app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -245,7 +245,9 @@ data class Patient(
             CASE
               WHEN
                 (BP.systolic >= 140 OR BP.diastolic >= 90
-                OR BloodSugar.bloodSugarValue IS NOT NULL
+                OR (BloodSugar.type = "random" AND BloodSugar.value >= 200)
+                OR (BloodSugar.type = "fasting" AND BloodSugar.value >= 300)
```

Comment:
There is a mistake here. The `fasting` and `random` values are switched.
Please refer the PRD and correct these.
Also update the tests accordingly.

Reply:
Fixed it 👍